Speed_Typing_Test.py

```diff
@@ -2,74 +2,80 @@
 from curses import wrapper
 import time
 import random
-
-
-def start_screen(stdscr):
-	stdscr.clear()
-	stdscr.addstr("Welcome to the Speed Typing Test!")
-	stdscr.addstr("\nPress any key to begin!")
-	stdscr.refresh()
-	stdscr.getkey()
-
-def display_text(stdscr, target, current, wpm=0):
-	stdscr.addstr(target)
-	stdscr.addstr(1, 0, f"WPM: {wpm}")
-
-	for i, char in enumerate(current):
-		correct_char = target[i]
-		color = curses.color_pair(1)
-		if char != correct_char:
-			color = curses.color_pair(2)
-
-		stdscr.addstr(0, i, char, color)
-
-def load_text():
-	with open("text.txt", "r") as f:
-		lines = f.readlines()
-		return random.choice(lines).strip()
-
-def wpm_test(stdscr):
-	target_text = "Hello this is some test text for this app!"
-	current_text = []
-	wpm = 0
-	start_time = time.time()
-	stdscr.nodelay(True)
-
-	while True:
-		time_elapsed = max(time.time() - start_time, 1)
-		wpm = round((len(current_text) / (time_elapsed / 60)) / 5)
-
-		stdscr.clear()
-		display_text(stdscr, target_text, current_text, wpm)
-		stdscr.refresh()
-
-		if "".join(current_text) == target_text:
-			stdscr.nodelay(False)
-			break
-
-		try:
-			key = stdscr.getkey()
-		except:
-			continue
-
-		if ord(key) == 27:
-			break
-
-		if key in ("KEY_BACKSPACE", '\b', "\x7f"):
-			if len(current_text) > 0:
-				current_text.pop()
-		elif len(current_text) < len(target_text):
-			current_text.append(key)
+from turtle import st
+
+class WpmTester:
```

Comment:
pouziti tridy, abys vsude nemusel predavat stdscr jako parameter, ale je to takto promena tridy.